tests/test_interpolation.py

```diff
@@ -450,19 +454,21 @@ def test_legacy_env_values_are_typed(
         assert c.my_key == expected
 
 
+# DEPRECATED: remove `recwarn` in 2.2 with the legacy env resolver
 @pytest.mark.parametrize("env_func", ["env", "oc.env"])
-def test_env_default_none(monkeypatch: Any, env_func: str) -> None:
+def test_env_default_none(monkeypatch: Any, recwarn: Any, env_func: str) -> None:
```

Comment:
If you add a trailing comma, black will format it as a list, which will allow you to place the comment in the right place.
```suggestion
def test_env_default_none(monkeypatch: Any, recwarn: Any, env_func: str,) -> None:
```

Reply:
Done in a6094e0